pages/index.tsx

```diff
@@ -1,6 +1,13 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from "react";
+import Head from "next/head";
+import Image from "next/image";
+import styles from "@/styles/Home.module.css";
+import NewMembersCard from "@/src/components/new-member";
+import { NEW_USER, NUM_MEMBERS_NUMBER } from "@/src/constants/AppConstants";
+
+const avatarImageSrc = require("../public/images/avatar.png");
```

Comment:
From next time onwards, please add a comment for informing about the temporary use of hard-coded or spoofed data.

Reply:
sure